src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -24,6 +25,7 @@ CHANGELOG
  * Made `BrowserKitAssertionsTrait` report the original error message in case of a failure
  * Added ability for `config:dump-reference` and `debug:config` to dump and debug kernel container extension configuration.
  * Deprecated `session.attribute_bag` service and `session.flash_bag` service.
+ * Wire `RedisTagAwareAdapter` for pools usings tags and Redis adapter
```

Comment:
Should be removed

Reply:
Looks like you removed the wrong one. The line for 5.1 should be move to the 5.2.0 section.

Reply:
Arf, ok I'm going to fix it, thank you :)

Reply:
Updated :) I changed the sentence to be more sticky with the final feature.